src/components/checkbox/checkbox.style.ts

```diff
@@ -12,7 +12,7 @@ import StyledValidationIcon from "../../__internal__/validations/validation-icon
 import baseTheme from "../../style/themes/base";
 import { ValidationProps } from "../../__internal__/validations";
 
-interface StyledCheckboxProps extends ValidationProps, MarginProps {
+export interface StyledCheckboxProps extends ValidationProps, MarginProps {
```

Comment:
It's only used in this file so this export is probably unnecessary

Reply:
looks like a leftover from my previous refactor approach, removed